### ISSUE TITLE
Blank Startup page

### ISSUE THREAD

**Issue by hamaney** [Microsoft/vscode#40352]
- VSCode Version: 1.19.0 (1.19.0)
- OS Version: 10.12.6 (16G1114)

Steps to Reproduce:

1. Download and install
2. Start the app

Reproduces without extensions: Yes (same as above produce the the same bug)

**Comment by ramya-rao-a** [comment]
Do you still see the issue when running `code --user-data-dir <path to some empty folder>`?
If the menu items are responding, then share any errors/callstacks from Help -> Toggle Developer Tools -> Console

**Comment by ramya-rao-a** [comment]
Most likely this is the same as #40329

@hamaney Can you check if you have `editor.tokenColorCustomizations` set to `null` in your settings file (you can find it at ~/Library/Application Support/Code/User/settings.json). If yes, then set it to `{}` instead and then try opening VS Code.

**Comment by hamaney** [comment]
@ramya-rao-a  yes I do see the issue when I run 
`code --user-data-dir <EmptyFolder>` and
`code --disable-gpu`

Here is the Console output:
```
index.js:5 TypeError: Cannot read property 'comments' of null
    at file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:2920:762
    at Array.forEach (native)
    at e.setCustomTokenColors (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:2920:735)
    at new e (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:2927:679)
    at t.create (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:90:103)
    at e._createInstance (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:1209:289)
    at e.createInstance (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:1208:560)
    at e.initServiceCollection (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:4516:741)
    at e.createContents (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:4511:118)
    at e.open (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:4518:113)
onError @ index.js:5
```

when I checked `editor.tokenColorCustomizations` i foun it was set to `null` in the settings file at `~/Library/Application Support/Code/User/settings.json`. After changing its value to `{}` it workd :)


Thanks

**Comment by ramya-rao-a** [comment]
duplicate of #40339 
The fix will be out in the recovery build of 1.19.1
